solr/core/src/test/org/apache/solr/handler/admin/CoreAdminHandlerActionTest.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.solr.handler.admin;
+
+import java.util.Map;
+import org.apache.solr.SolrTestCaseJ4;
+import org.apache.solr.common.SolrException;
+import org.apache.solr.common.params.CoreAdminParams;
+import org.apache.solr.common.util.NamedList;
+import org.apache.solr.common.util.SimpleOrderedMap;
+import org.apache.solr.handler.admin.CoreAdminHandler.CallInfo;
+import org.apache.solr.handler.admin.CoreAdminHandler.CoreAdminOp;
+import org.apache.solr.response.SolrQueryResponse;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CoreAdminHandlerActionTest extends SolrTestCaseJ4 {
+
+  private static CoreAdminHandler admin = null;
+
+  @BeforeClass
+  public static void beforeClass() throws Exception {
+
+    setupNoCoreTest(createTempDir(), null);
+
+    admin = new CoreAdminHandler(h.getCoreContainer());
+
+    admin.registerCustomActions(
+        Map.of(
+            "action1",
+            new CoreAdminHandlerTestAction1(),
+            "action2",
+            new CoreAdminHandlerTestAction2()));
+  }
```

Comment:
I doesn't come up with some simple idea of how to make core container loader to link my test action classes. It seems it is possible with applying some heavy staff or by mocking it with Mockito, I think it is unreasonably complex for the case trying make them "true" loaded. So I ended up with registering them manually.

Reply:
Just added https://github.com/apache/solr/pull/931/commits/60805de04b06118ba1b5d49332d41b6dad1e5c07 commit, what do you think?

Reply:
Thank you for the update! I tried exachtly the same thing before, but when I run test locally it fails complaining about loader is unable to load the action class. I tried yours update and, magic, it passed! Arghhh :)